core/fnv1a.js

```diff
@@ -38,7 +38,7 @@ module.exports = class FNV1a {
 
 	digest(encoding) {
 		encoding = encoding || 'binary';
-		let buf = new Buffer(4);
+		let buf = Buffer.alloc(4);
```

Comment:
I should have probably been using a const here. FWIW, I think we can use `allocUnsafe()` in cases such as this where we *know* we're about to overwrite the entire chunk of memory anyway (https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe)